CONTRIBUTING.md

```diff
@@ -47,11 +54,15 @@ const {
   run: { scheduleOnce }
 } = Ember;
 ```
-* Order the imported variables in order of appearance, or other logical order.
+ * Order the imported variables in order of appearance, or other logical order.
 
 ## Converting components from Angular Material
 
-* Use an ember-paper name for components. Convert angular elements attributes to ember boolean properties.
+* **Stylesheets.** Use the Angular Material stylesheets as-is. Replicate Angular markup in `ember-paper`.
+
+* **Naming.** Use an ember-paper name for components.
+
+* **Attributes.** Convert angular elements attributes to ember boolean properties.
```

Comment:
Remove `boolean`. 
`{{paper-tabs someProp="example"}}` can exist.

ember "boolean" is equivalent to the presence/absence of an element attribute in angular.